### ISSUE TITLE
Resolve error: unable to load resolver "node". (import/no-duplicates)

### ISSUE THREAD

**Issue by binarykitchen** [ricardofbarros/linter-js-standard#198]
Always seeing this warning in v3.9.3 under Atom

```
Resolve error: unable to load resolver "node". (import/no-duplicates)
```

in the project, using standardjs 10.0.3 ... no idea how to fix this warning.

**Comment by sonicdoe** [comment]
Does this warning only occur in Atom or also when running `standard` from the command line?

**Comment by binarykitchen** [comment]
sometimes only in atom, sometimes both, very unreliable, so cannot answe

**Comment by sonicdoe** [comment]
Your issue sounds similar to https://github.com/standard/standard/issues/848. It seems to be caused by the structure of the `node_modules` directory so you could try re-installing `standard` with the newest version of npm.
